recipes/lapack95/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "lapack95" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://www.netlib.org/lapack95/lapack95.tgz
+  sha256: 55711e5b1c5ddc9c7dd1d3ea15c37fc5b400783ac6ea573a75e2b1358c2c479b
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+  host:
+    - libblas
+    - liblapack
+  run:
+    - libgfortran
```

Comment:
This shouldn’t be needed as the Fortran compiler will already add it’s runtime libraries as dependencies

Reply:
Addressed via https://github.com/conda-forge/staged-recipes/pull/14212/commits/02def9f4d92b182dbd2ed82c4db29bbdf4d25d59